arm-monitor/2017-03-01/swagger/activityLogAlerts_API.json

```diff
@@ -0,0 +1,451 @@
+                    "description": "The conditon that will cause this alert to activate."
+                },
+                "actions": {
+                    "$ref": "#/definitions/ActivityLogAlertActionList",
+                    "description": "The actions that will activate when the condition is met."
+                },
+                "description": {
+                    "type": "string",
+                    "description": "A description of this activity log alert."
+                }
+            },
+            "required": [
+                "scopes",
+                "condition",
+                "actions"
+            ]
+        },
+        "ActivityLogAlertAllOfCondition": {
+            "properties": {
+                "allOf": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/ActivityLogAlertLeafCondition"
+                    },
+                    "description": "The list of activity log alert conditions."
+                }
+            },
+            "description": "An Activity Log alert condition that is met when all its member conditions are met.",
+            "required": [
+                "allOf"
+            ]
+        },
+        "ActivityLogAlertLeafCondition": {
+            "properties": {
+                "field": {
+                    "type": "string",
+                    "description": "The name of the field that this condition will examine. The possible values for this field are (case-insensitive): 'resourceId', 'category', 'caller', 'level', 'operationName', 'resourceGroup', 'resourceProvider', 'status', 'subStatus', 'resourceType', or anything beginning with 'properties.'."
+                },
+                "equals": {
+                    "type": "string",
+                    "description": "The field value will be compared to this value (case-insensitive) to determine if the condition is met."
+                }
+            },
+            "description": "An Activity Log alert condition that is met by comparing an activity log field and value.",
+            "required": [
+                "field",
+                "equals"
+            ]
+        },
+        "ActivityLogAlertActionList": {
+            "description": "A list of activity log alert actions.",
+            "properties": {
+                "actionGroups": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/ActivityLogAlertActionGroup"
+                    },
+                    "description": "The list of activity log alerts."
+                }
+            }
+        },
+        "ActivityLogAlertActionGroup": {
+            "description": "A pointer to an Azure Action Group.",
+            "properties": {
+                "actionGroupId": {
+                    "type": "string",
+                    "description": "The resourceId of the action group. This cannot be null or empty."
```

Comment:
Is there any format of `actionGroupId`? If so we can detect it from regex on client side?